src/main/java/com/github/awwkoala/selenium/util/WebDriverProvider.java

```diff
@@ -0,0 +1,13 @@
+package com.github.awwkoala.selenium.util;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebDriverProvider {
+
+  public WebDriver getChromeDriver() {
+    System.setProperty("webdriver.chrome.driver", "/opt/WebDriver/bin/chromedriver");
```

Comment:
Optional:
- currently you have to manually manage your chrome driver on your machine and explicitly setup path to it in tests.
- you can read about webDriverManager lib which provides you chromeDriver out of the box https://medium.com/@mustafayol/ultimate-helper-of-browser-tests-bonigarcias-webdriver-manager-b12fe0ea674b